pages/create-bill/index.js

```diff
@@ -0,0 +1,186 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import Link from "next/link";
+import uuid from "react-uuid";
+import BillDetails from "@/components/BillDetails";
+
+export default function BillForm() {
+  const [title, setTitle] = useState("");
+  const [amount, setAmount] = useState("");
+  const [participants, setParticipants] = useState([
+    {
+      name: "",
+      id: uuid(),
+      paid: 0,
+    },
+  ]);
+  const [results, setResults] = useState([]);
+
+  // function validateTitle(title) {
```

Comment:
is that out commented code for your next userStory ?

Reply:
> maybe you should round the numbers up to the 2 decimal place because you get endless decimal places when the corresponding amount is entered. Also, I'm not quite clear how it works when some have paid more than others. It says that those who have paid more get something and those who have paid less owe something but this would have to be divided again on all. But maybe you already have that in planning. Otherwise this works wonderfully

thanks a lot! i just looked up and will round the numbers, and i am also working on some form validation so the inputs are a bit more controlled, but essentially the paid amounts are compared to the even split of the total, so the result is the remaining amount of that for everyone. thanks for reviewing!